java-shared-config/pom.xml

```diff
@@ -572,14 +572,9 @@
       </activation>
       <properties>
         <!-- default config values -->
-        <docletName>java-docfx-doclet-1.9.0</docletName>
-        <docletPath>${env.KOKORO_GFILE_DIR}/${docletName}.jar</docletPath>
```

Comment:
I thought this docletPath specifies the JAR file that has implementation of java-docfx-doclet. Don't we still need this?

Reply:
We're setting the `docletPath` directly in the script that publishes the Cloud RAD tarballs here: https://source.corp.google.com/piper///depot/google3/cloud/sdk/client_libraries/kokoro/java/googleapis/java-cloud-bom/release/publish_javadoc17.sh;l=45
so I don't think this is needed.